game.controller.amstrad.joystick/resources/layout.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<layout label="30000" image="layout.png">
+  <category name="face" label="35074">
+    <button name="button1" type="digital" label="30001"/>
+    <button name="button2" type="digital" label="30002"/>
+    <button name="up" type="digital" label="30003"/>
+    <button name="right" type="digital" label="30004"/>
+    <button name="down" type="digital" label="30005"/>
+    <button name="left" type="digital" label="30006"/>
+  </category>
+  <category name="hardware" label="35108">
+    <button name="run" type="digital" label="30007"/>
+    <button name="vkbtoggle" type="digital" label="30008"/>
+    <button name="return" type="digital" label="30009"/>
+    <button name="mousetoggle" type="digital" label="30010"/>
+    <button name="cat" type="digital" label="30011"/>
+    <button name="stat" type="digital" label="30012"/>
+    <button name="reset" type="digital" label="30013"/>
+    <button name="tape" type="digital" label="30014"/>
+</layout>
```

Comment:
category end tag is missing

Reply:
@garbear If you fix that it would be awesome if you could rebase, as rechi fixed a problem in the ci runner.